flink-tests/src/test/java/org/apache/flink/test/checkpointing/AutoRescalingITCase.java

```diff
@@ -513,7 +515,7 @@ public void testCheckpointRescalingPartitionedOperatorState(
         // wait until the operator handles some data
         StateSourceBase.workStartedLatch.await();
 
-        waitForOneMoreCheckpoint(jobID, cluster.getMiniCluster());
+        waitForNewCheckpoint(jobID, cluster.getMiniCluster());
```

Comment:
AFAIU, we don't need to change it here.

Reply:
Same with this comment: https://github.com/apache/flink/pull/24246/files#r1474917357